0001/README.md

```diff
@@ -0,0 +1,400 @@
+not open for discussion.)_
+
+Goals
+
+* Explain what a Proposal is, how and when to use it
+* Share the Proposal lifecycle
+* Be the first Proposal for the project
+
+Non-Goals
+
+* Offering all of the automation, scripts and bots described as part of the Proposal
+  lifecycle
+* Converting the already open PR against the
+    [proposal](https://github.com/tinkerbell/proposals) repo to this new format
+
+## Content
+
+_(This is the main section. It contains the core of the discussion, the design
+you are proposing, the changes this imposes and the new features it provides.
+This should include a high-level overview of how this proposal can be
+implemented.)_
+
+### Why a Proposal is needed and what it looks like
+
+Discussing and deciding how and why a feature or a piece of code should or
+should not be written is an important part of the engineering process.
+
+As an open source project, Tinkerbell needs to have a public place where
+community members, contributors can share their ideas in detail, asking for
+help, feedback and more in general to spot problems addressable as early as
+possible.
+
+A maintainer can ask contributors to write an Proposal in order to open a discussion
+about a feature that looks trivial, helping contributors to sit down and think
+about the design of the request or the code they will build. It is a way to
+translate Slack discussions to something that won’t get lost for example.
+
+Currently, we have a repo called
+[proposal](https://github.com/tinkerbell/proposals) that I would like to rename
+to Proposal. Every Proposal will have its own directory with a markdown file inside.
+
+### Metadata
+
+The Proposal is written in markdown and includes metadata in the header of the file:
+
+* `ID`: identifies the Proposal
+* `Title`: title of the Proposal
+* `Status`: description of the current state for the Proposal
+* `Authors`: in the format of a list of `Name <email@emai.com>`
+
+Here is a complete example:
+
+```yaml
+---
+id: 0001
+title: Request for discussion HOWTO
+pr: 2
+status: ideation
+authors: Gianluca Arbezzano <gianarb92@gmail.com>
+---
+```
+
+### Lifecycle
+
+I don’t want to rewrite the lifecycle here because the one in use by [Oxide
+Computer](https://oxide.computer/blog/rfd-1-requests-for-discussion/#rfd-life-cycle)
+looks familiar and is functional for our purpose. They wrote the lifecycle in
+detail and, for now, we will follow it as it is.
+
+What follows here is copy/pasted from ["RFD 1 Requests for
+Discussion"](https://oxide.computer/blog/rfd-1-requests-for-discussion/) and all
+the credit goes to [Oxide Computer](https://oxide.computer).
+
+~~~markdown
+
+### RFD Metadata and State
+
+At the start of every RFD document, we'd like to include a brief amount of metadata. The metadata format is based on the python-markdown2 metadata format. It'd look like:
+
+---
+authors: Andy Smith <andy@example.computer>, Neal Jones <neal@example.computer>
```

Comment:
I'd suggest we use at least one predominantly female name here, or gender neutral / placeholder names instead of 2 predominantly masculine.